dev/src/index.ts

```diff
@@ -1007,6 +1025,21 @@ export class Firestore {
       });
   }
 
+  /**
+   * Terminates the Firestore client and closes all open streams.
+   * @return A Promise that resolves when the client is terminated.
+   */
+  async terminate(): Promise<void> {
```

Comment:
Please remember to add this to our public types.

Reply:
Done.